docs/src/content/docs/cookbook/dexes/stonfi.mdx

```diff
@@ -5,10 +5,362 @@ sidebar:
+        // In real-world scenarios, it's more reliable to calculate this address on-chain or save it during initialization to ensure accuracy and trustworthiness.
+        let myJettonWalletAddress = msg.myJettonWalletAddress;
+
+        let offerAmount = 1000000; // the quantity of jettons being transferred for liquidity provisioning
+
+        let forwardPayload = ProvideLP{
+            otherTokenWallet: RouterProxyTonWallet,
+            refundAddress: myAddress(),
+            excessesAddress: myAddress(),
+            deadline: now() + 1000,
+            additionalData: ProvideLPAdditionalData{
+                minLpOut: 1,
+                receiverAddress: myAddress(),
+                bothPositive: false, // false means single side
+            }
+        };
+
+        send(SendParameters{
+            to: myJettonWalletAddress,
+            value: SingleSideProvideLpJettonGas,
+            body: JettonTransfer{
+                queryId: 42,
+                amount: offerAmount,
+                destination: RouterAddress,
+                responseDestination: myAddress(),
+                forwardTonAmount: SingleSideProvideLpJettonGasFwd,
+                forwardPayload: forwardPayload.toCell(),
+            }.toCell(),
+        });
+    }
+}
+```
+
+### TON Deposit
+
+```tact
+message(0x01f3835d) ProxyTonTransfer {
+    queryId: Int as uint64 = 0;
+    tonAmount: Int as coins;
+    refundAddress: Address;
+    forwardPayload: Cell?;
+}
+
+const RouterProxyTonWallet: Address = address("kQBbJjnahBMGbMUJwhAXLn8BiigcGXMJhSC0l7DBhdYABhG7"); // Router's pTON wallet Address
+const RouterJettonWallet: Address = address("kQAtX3x2s-wMtYTz8CfmAyloHAB73vONzJM5S2idqXl-_5xK"); // Router's Jetton Wallet Address
+
+contract Sample {
+    receive() {}
+    receive("provide-lp-ton") {
+        let offerAmount = 1000000; // the quantity of TON being transferred for liquidity provisioning
+
+        let forwardPayload = ProvideLP{
+            otherTokenWallet: RouterJettonWallet,
+            refundAddress: myAddress(),
+            excessesAddress: myAddress(),
+            deadline: now() + 1000,
+            additionalData: ProvideLPAdditionalData{
+                minLpOut: 1,
+                receiverAddress: myAddress(),
+                bothPositive: false, // false means single side
+            }
+        };
+
+        send(SendParameters{
+            to: RouterProxyTonWallet,
+            value: SingleSideProvideLpTonGas + TonTransferGas + offerAmount,
+            body: ProxyTonTransfer{
+                queryId: 42,
+                tonAmount: offerAmount,
+                refundAddress: myAddress(),
+                forwardPayload: forwardPayload.toCell(),
+            }.toCell(),
+        });
+    }
+}
+```
+
+### Withdraw liquidly
+
+Liquidly is withdrawn from a pool by burning lp tokens, a user then receives both pool tokens based on the current exchange rate.
```

Comment:
Probably has to be modified in the same way similar sentences are expressed in DeDust cookbook